src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -212,6 +216,9 @@ def resolve_mount(self, location: str | None) -> str | None:
         if not location.startswith('dbfs:'):
             return location  # not a mount, save some cycles
         for mount in self._mounts_snapshot:
+            # Skip the root mount
```

Comment:
Why is needs the root mount be skipped? We want these still to be replace, or not?

Reply:
Nope, we don't care about the actual location of the DBFS root tables. Because we don't want to create individual external locations, rather a single one for all dbfs root tables.